### ISSUE TITLE
dry-configurable 0.13 breaks this gem

### ISSUE THREAD

**Issue by FireFart** [waiting-for-dev/warden-jwt_auth#34]
dry-configurable 0,13 was released:
https://github.com/dry-rb/dry-configurable/releases/tag/v0.13.0

This breaks the current version of the gem:
```
NoMethodError:
  undefined method `each_with_object' for nil:NilClass
# ./vendor/bundle/ruby/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:23:in `symbolize_keys'
# ./vendor/bundle/ruby/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:62:in `block in <module:JWTAuth>'
# ./vendor/bundle/ruby/2.7.0/gems/dry-configurable-0.13.0/lib/dry/configurable/setting.rb:147:in `evaluate'
# ./vendor/bundle/ruby/2.7.0/gems/dry-configurable-0.13.0/lib/dry/configurable/setting.rb:79:in `value'
# ./vendor/bundle/ruby/2.7.0/gems/dry-configurable-0.13.0/lib/dry/configurable/config.rb:77:in `block in values'
# ./vendor/bundle/ruby/2.7.0/gems/dry-configurable-0.13.0/lib/dry/configurable/settings.rb:49:in `each'
# ./vendor/bundle/ruby/2.7.0/gems/dry-configurable-0.13.0/lib/dry/configurable/settings.rb:49:in `each'
# ./vendor/bundle/ruby/2.7.0/gems/dry-configurable-0.13.0/lib/dry/configurable/config.rb:77:in `map'
# ./vendor/bundle/ruby/2.7.0/gems/dry-configurable-0.13.0/lib/dry/configurable/config.rb:77:in `values'
# ./vendor/bundle/ruby/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:20:in `jwt'
# ./config/initializers/02_devise.rb:308:in `block in <top (required)>'
# ./vendor/bundle/ruby/2.7.0/gems/devise-4.8.0/lib/devise.rb:307:in `setup'
# ./config/initializers/02_devise.rb:11:in `<top (required)>'
```

I would suggest to lock down the dry-configurable version in the Gemfile or make it compatible with the breaking changes

**Comment by waiting-for-dev** [comment]
Fixed on master. I'll release a new version as soon as possible.

**Comment by FireFart** [comment]
Awesome thanks!

**Comment by cjnahine** [comment]
Any updates on new version regarding this fix?

`C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:19:in `<module:Warden>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:19:in `<module:Warden>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:19:in `<module:Warden>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:19:in `<module:Warden>' [dry-configurable] passing a constructor as a block is deprecated and will be removed in the next major version
Provide a `constructor:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:19:in `<module:Warden>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:19:in `<module:Warden>' [dry-configurable] passing a constructor as a block is deprecated and will be removed in the next major version
Provide a `constructor:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:19:in `<module:Warden>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:19:in `<module:Warden>' [dry-configurable] passing a constructor as a block is deprecated and will be removed in the next major version
Provide a `constructor:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:19:in `<module:Warden>' [dry-configurable] passing a constructor as a block is deprecated and will be removed in the next major version
Provide a `constructor:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:28:in `<module:Devise>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:28:in `<module:Devise>' [dry-configurable] passing a constructor as a block is deprecated and will be removed in the next major version
Provide a `constructor:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:28:in `<module:Devise>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:28:in `<module:Devise>' [dry-configurable] passing a constructor as a block is deprecated and will be removed in the next major version
Provide a `constructor:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:28:in `<module:Devise>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:28:in `<module:Devise>' [dry-configurable] passing a constructor as a block is deprecated and will be removed in the next major version
Provide a `constructor:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:28:in `<module:Devise>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:28:in `<module:Devise>' [dry-configurable] passing a constructor as a block is deprecated and will be removed in the next major version
Provide a `constructor:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:28:in `<module:Devise>' [dry-configurable] default value as positional argument to settings is deprecated and will be removed in the next major version
Provide a `default:` keyword argument instead

C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:28:in `<module:Devise>' [dry-configurable] passing a constructor as a block is deprecated and will be removed in the next major version
Provide a `constructor:` keyword argument instead

rake aborted!
NoMethodError: undefined method `each_with_object' for nil:NilClass
E:/TOMO/advanceme-api/config/initializers/devise.rb:20:in `block in <main>'
E:/TOMO/advanceme-api/config/initializers/devise.rb:11:in `<main>'
E:/TOMO/advanceme-api/config/environment.rb:5:in `<main>'
`

**Comment by bnjtgly** [comment]
Hi, I encountered the same issue. Although I saw the updates on the Repo, when will the new version release for the fix?

**Comment by stephenarifin** [comment]
I am also running into this issue. Hoping for the fix to come soon!

**Comment by lsuman83** [comment]
Issue seems to be with hash parameter passed into the symbolize keys method. Having the same problem when trying to use rails generator and rails server

**Comment by waiting-for-dev** [comment]
Released on v0.6.0
